src/api/schema/components/mod.rs

```diff
@@ -28,9 +31,8 @@ pub enum Component {
     Sink(sink::Sink),
 }
 
-#[derive(Enum, Copy, Clone, Eq, PartialEq)]
-/// Type of component
-pub enum ComponentType {
+#[derive(Enum, Copy, Clone, Eq, PartialEq, Ord, PartialOrd)]
+pub enum ComponentKind {
```

Comment:
This was renamed from `ComponentType` -> `ComponentKind`, to match the lexicon we're using in official docs.

`component_type` as a field, remains the specific end-type... e.g. `file`, `blackhole`, etc.